docs/content/concepts/webserver/graphql.mdx

```diff
@@ -137,7 +112,11 @@ For example, the following query will return all failed runs:
 
 ```shell
 query FilteredRunsQuery {
-  runsOrError(filter: { statuses: [FAILURE] }) {
+  runsOrError(
+    filter: { statuses: [FAILURE] }
+    cursor: "7fd2e5ef-5591-43db-be15-1ebbbbed8bb5"
+    limit: 10
+  ) {
```

Comment:
maybe
```
query FilteredRunsQuery($cursor: String) {
  runsOrError(
    filter: { statuses: [FAILURE] }
    cursor: $cursor
    limit: 10
  ) {
  ```
  
  instead of random id

Reply:
Sure - I just grabbed the same one from the previous paginated example.